test/integration/targets/lookup_properties/test_lookup_properties.yml

```diff
@@ -9,20 +9,20 @@
         test2:            "{{lookup('ini', 'value2            type=properties  file=lookup.properties')}}"
         test_dot:         "{{lookup('ini', 'value.dot         type=properties  file=lookup.properties')}}"
         field_with_space: "{{lookup('ini', 'field.with.space  type=properties  file=lookup.properties')}}"
-    - debug: var={{item}}
+    - debug: var=item
```

Comment:
this is relying on double interpolation to test dispay of test2, test_dot, not of item